rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.4.8-release-notes.adoc

```diff
@@ -0,0 +1,44 @@
+= RPA Builder 6.4.8 Release Notes
+
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== September 29, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* Enhanced logging for Outlook with OAuth.
+* You can now configure a custom timeout for REST Call. 
```

Comment:
To keep the consistency between bullets, you can say:

"Configurable custom timeout for REST Call"
"Tomcat dependency upgraded to version 9.0.80."